fmriprep/cli/run.py

```diff
@@ -282,6 +288,26 @@ def main():
     if opts.reports_only:
         sys.exit(int(retcode > 0))
 
+    # Sentry tracking
+    if not opts.notrack:
+        try:
+            from raven import Client
+            dev_user = bool(int(os.getenv('FMRIPREP_DEV', 0)))
+            msg = 'fMRIPrep running%s' % (int(dev_user) * ' [dev]')
+            client = Client(
+                'https://d5a16b0c38d84d1584dfc93b9fb1ade6:'
+                '21f3c516491847af8e4ed249b122c4af@sentry.io/1137693',
+                release=__version__)
+            client.captureMessage(message=msg,
+                                  level='debug' if dev_user else 'info',
+                                  tags={
+                                      'run_id': run_uuid,
+                                      'npart': len(subject_list),
+                                      'type': 'ping',
+                                      'dev': dev_user})
```

Comment:
Are we capturing all of the command line arguments?

Reply:
If we are going to capture args, for data, output and work directories, it would be good to replace all but the last path element with "..." (i.e. `/path/to/bids/ds000XXX` becomes `.../ds000XXX`). It's unimportant to us, and it's information about someone's system that just doesn't need to be shared.

Reply:
Sentry captures sys.argv by default. I have to read the Data filtering module they have to do what @effigies suggests